<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-HJ1GF5N3" failures="1" tests="3" name="Test" time="0.113" errors="0" timestamp="2024-04-05T13:50:44 EDT">
  <testcase classname="testNg_package.Skip_testcase" name="test1" time="0.013"/>
  <testcase classname="testNg_package.Skip_testcase" name="test2" time="0.002">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase classname="testNg_package.Skip_testcase" name="test3" time="0.021">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testNg_package.Skip_testcase.test3(Skip_testcase.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testNg_package.Skip_testcase" name="bar" time="0.0">
    <ignored/>
  </testcase> <!-- bar -->
</testsuite> <!-- Test -->
